Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'cocoapods', '1.16.2'
 gem 'cocoapods-trunk', '1.6.0'
 gem 'danger', '9.5.1'
 gem 'rest-client', '2.1.0'
-gem 'nokogiri', '1.16.7'
+gem "nokogiri", ">= 1.18.4"
```

Comment:
Hmm I wonder if we should just not indicate the version here and rely on the lock file to keep track of the version used... NABD though.

Reply:
I like it. I will remove the version here